Sources/Danger/BitBucketCloud.swift

```diff
@@ -193,7 +193,7 @@ extension BitBucketCloud {
     public struct Comment: Decodable, Equatable {
         public struct Inline: Decodable, Equatable {
             public let from: Int?
-            public let to: Int? // swiftlint:disable:this identifier_name
+            public let to: Int? 
```

Comment:
<!--
  0 failure: 
  2 warning:  Lines should not ..., Variable name sho...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/BitBucketCloud.swift;
  Line: 196;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)
- :warning: Variable name should be between 3 and 40 characters long: 'to' (`identifier_name`)